AboutUs/index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>About Us</title>
+</head>
+<header>
+    <div class="header-container">
+        <div class="logo">
+          <img src="/images/TechGirls.jpg" alt="TechGirlsHub Logo">
+          <h1>TECHGIRLSHUB</h1>
+        </div>
+        <nav>
+          <ul>
+            <li><a href="#">About Us</a></li>
```

Comment:
please include the url links,  and  Feedback Rating link - url should be feedbackRating.php